pypesto/petab/importer.py

```diff
@@ -171,6 +160,15 @@ def from_yaml(
             model_name=model_name,
         )
 
+    def validate_inner_options(self):
+        """Validate the inner options."""
+        for key in self.inner_options:
+            if (
+                key not in OPTIMAL_SCALING_OPTIONS
```

Comment:
combine checks by checking membership in `OPTIMAL_SCALING_OPTIONS + SPLINE_APPROXIMATION_OPTIONS` or `(*OPTIMAL_SCALING_OPTIONS,*SPLINE_APPROXIMATION_OPTIONS)`?